### PR TITLE
Url shortening

### PR DESCRIPTION
**Pull request by GHkrishna**
Adds functionality for URL shortening

### CODE REVIEW

File: apps/verification/src/verification.service.ts [hunk 1]
```diff
@@ -419,6 +425,47 @@ export class VerificationService {
     }
   }
 
+  async storeVerificationObjectAndReturnUrl(storeObj: string, persistent: boolean): Promise<string> {
+    //nats call in agent-service to create an invitation url
+    const pattern = { cmd: 'store-object-return-url' };
+    const payload = { persistent, storeObj };
+
+    // try {
```
Comment (vivekayanworks) [inline apps/verification/src/verification.service.ts:433]: Remove unused commented code

File: apps/issuance/src/issuance.service.ts [hunk 2]
```diff
@@ -260,6 +263,48 @@ export class IssuanceService {
...
   }
 
+  async storeIssuanceObjectReturnUrl(storeObj: string): Promise<string> {
+    // Set default to false, since currently our invitation are not multi-use
+    const persistent: boolean = false;
+    //nats call in agent-service to create an invitation url
+    const pattern = { cmd: 'store-object-return-url' };
+    const payload = { persistent, storeObj };
+
+    // try {
```
Comment (vivekayanworks) [inline apps/issuance/src/issuance.service.ts:273]: Remove commented code

File: apps/verification/src/verification.service.ts (hunk 1 above)
Comment (GHkrishna) [reply apps/verification/src/verification.service.ts]: Resolved

File: apps/issuance/src/issuance.service.ts (hunk 2 above)
Comment (GHkrishna) [reply apps/issuance/src/issuance.service.ts]: Resolved

**Review by sairanjit — APPROVED** [review]
LGTM

**Review by ankita-p17 — APPROVED** [review]

**Pull request merged**